docs/release-notes/NuGet-6.10.md

```diff
@@ -0,0 +1,178 @@
+---
+title: NuGet 6.10 Release Notes
+description: Release notes for NuGet 6.10 including new features, bug fixes, and DCRs.
+author: kartheekp-ms
+ms.author: kartheekp-ms
+ms.date: 5/13/2024
+ms.topic: conceptual
+---
+
+# NuGet 6.10 Release Notes
+
+NuGet distribution vehicles:
+
+| NuGet version | Available in Visual Studio version | Available in .NET SDK(s) |
+|:---|:---|:---|
+| [**6.10**](https://nuget.org/downloads) | [Visual Studio 2022 version 17.10](https://visualstudio.microsoft.com/downloads/) | [8.0.3xx](https://dotnet.microsoft.com/download/dotnet/8.0)<sup>1</sup> |
+
+<sup>1</sup> Installed with Visual Studio 2022 with any .NET workload
+
+## Summary: What's New in 6.10
+
+* Add event tracing to restore to make it possible to measure performance - [#13274](https://github.com/NuGet/Home/issues/13274)
+
+* Warn when vulnerabilities are detected during packages.config restore in CLI scenarios. - [#13253](https://github.com/NuGet/Home/issues/13253)
+
+* Read auditSources from nuget.config files - [#13211](https://github.com/NuGet/Home/issues/13211)
+
+* [Feature]: add command dotnet config set - [#12476](https://github.com/NuGet/Home/issues/12476)
```

Comment:
Is it only set, or do we have dotnet nuget config in general?

Feels like we're missing a general issue for `dotnet nuget config`.

I also feel like this shipped some time ago, but maybe I'm missing something. 

I found https://github.com/NuGet/Home/issues/12469. @martinrrm Can you help us figure what the status is here?

Reply:
This was merged to main in January https://github.com/NuGet/NuGet.Client/pull/5549. I believe Dainiz started the docs but I can finish that (tracking: https://github.com/NuGet/Home/issues/13398)

Reply:
So January would be this shipped in 6.9 correct?

Reply:
Yeah it should have been mentioned in 6.9 but I don't think we did. Should we delete it from here?

Reply:
Tricky :D Not sure tbh. 

From a release notes perspective, I feel like 6.9 should contain the fact that this shipped.

Looks like we have a docs issue for documenting dotnet nuget config, so we should try to have that done soon.

What's the right way to get visibility for the feature? Should we just have it be part of our blog, but not part of our release notes? 

cc @JonDouglas @NuGet/nuget-client Thoughts?

Reply:
Let's just ship it in 6.10. It wasn't mentioned in 6.9 release notes that i can tell. Given it was caught by tooling here, it should be fine to just highlight it in this blog?

Reply:
Pretend like it's shipping in 6.10. 

I'm fine with that. 

We should add that detail in the issue, but have it marked as 6.10 as far as release notes gotes.

Reply:
> We should add that detail in the issue, but have it marked as 6.10 as far as release notes gotes.

Done. Nikolche. https://github.com/NuGet/Home/issues/12476#issuecomment-2112985657